taxonomies/nutrients.txt

```diff
@@ -3594,7 +3600,7 @@ or:ସୋଡ଼ିଅମ
 pa:ਸੋਡੀਅਮ
 pi:सोडियम
 pl:Sód
-pt:Sódio
+pt:Sódio, Na+, Na
```

Comment:
Is there any product with "Na+" specified as a nutrient?

Reply:
Is shown in this label in the same water bottle:
![nutrition_pt 43 full_2](https://user-images.githubusercontent.com/80917261/115226791-29017c80-a0ff-11eb-96f4-7dc583867b3e.jpg)